app/controllers/projects_controller.rb

```diff
@@ -12,7 +12,10 @@ class ProjectsController < ApplicationController
   def index
     @projects = {}
 
-    projects_joined = policy_scope(Project).preload(:tag_group)
+    projects_joined = policy_scope(Project)
+                      .joins(:teams)
+                      .where(teams: { id: current_team.id })
```

Comment:
You can pass the object itself and Rails will get the id for you:

```suggestion
                      .where(teams: { id: current_team })
```